src/EPR.Calculator.API/Exporter/CalcResultsExporter.cs

```
@@ -23,55 +23,46 @@ public class CalcResultsExporter : ICalcResultsExporter<CalcResult>
         public string Export(CalcResult results)
         {
             var csvContent = new StringBuilder();
-            LoadCalcResultDetail(results, csvContent);
-            if (results.CalcResultLapcapData != null)
+            if (results == null)
             {
-                PrepareLapcapData(results.CalcResultLapcapData, csvContent);
+                return csvContent.ToString();
```

Probably you can throw Exception here rather than return Empty string

---

done